tool/tctl/common/tctl.go

```diff
@@ -174,6 +174,9 @@ func Run(commands []CLICommand) {
 
 	client, err := authclient.Connect(ctx, clientConfig)
 	if err != nil {
+		if utils.IsUntrustedCertErr(err) {
+			err = trace.WrapWithMessage(err, utils.SelfSignedCertsMsg)
+		}
```

Comment:
Added this so `tctl` with self-signed certs prints the self-signed certs message if they don't use `--insecure`